packages/manager/apps/pci-object-storage/public/translations/common/Messages_de_DE.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
you cam import common translations from pci-common using "pci-common" key, no need to copy them in the app